src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.spec.ts

```diff
@@ -103,11 +113,45 @@ describe(commands.TEAMS_REPORT_DEVICEUSAGEDISTRIBUTIONUSERCOUNTS, () => {
     assert.equal(actual, true);
   });
 
+  it('passes validation on valid \'text\' output', () => {
```

Comment:
I don't think we need this test here since we're not introducing a new output type and json and text are already covered in the base class.

Reply:
Done. This is fixed